.github/workflows/codeql-analysis.yml

```diff
@@ -2,12 +2,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, 'b[0-9].[0-9]+' ]
```

Comment:
Yeah, I guess that's what we've been doing, although I never liked marking *release branches* with a `b` prefix.

Reply:
> I never liked marking _release branches_ with a `b` prefix.

Given that Git supports much fancier refs (e.g., 'releases/0.73`), it does seem sort of minimalist...but I'm just going with what we've got (that is, we're not going to rename the old branches, and, at this point, we're unlikely to adopt a new scheme for the new ones...are we?).

(Naming the branches that way bothers be less than it might, given that the actual _releases_ have individual tags, not just branches.)

Of course, I now need to back-port this change to `b0.73`, and, presumably, `b0.72`, and possibly farther...I don't know....

Reply:
Right; I'm accustomed to, and would much prefer, names like `release/0.73` or `rel/0.73`, but, as you say, it is what it is and that's all that it is...